Pokemon.java

```diff
@@ -0,0 +1,135 @@
+import java.util.Arrays;
+import java.util.List;
+
+abstract class Pokemon {
+    private String name;
+    private int level;
+    private int hp;
+    private final int maxHP;
+    private int xp;
+    private final int defence;
+    private final int attack;
+    private String food;
+    private final int accuracy;
+    private String[] pokemonType;
```

Comment:
Ik denk niet dat een String[] hier de beste oplossing is. Ik denk dat het beter is om een 'String mainType' en een 'String subType' te maken. Je wilt namelijk geen pokemon maken dat 25 types heeft.